### PR TITLE
Fix x5t issue in JWKS endpoint

### PR DESCRIPTION
**Pull request by lashinijay**
**Before this fix** 
```
"x5t#S256": "AuYhelj-jvDRkeMPas9Ezwj0e0wYxsZKvfLUh7SMCgA"
```

**After the fix**
```
"x5t#S256": "MDJlNjIxN2E1OGZlOGVmMGQxOTFlMzBmNmFjZjQ0Y2YwOGY0N2I0YzE4YzZjNjRhYmRmMmQ0ODdiNDhjMGEwMA"
```

**Based 64 decoded value after the fix**
```
02e6217a58fe8ef0d191e30f6acf44cf08f47b4c18c6c64abdf2d487b48c0a00
```
### Related Issues

- Public Issue: https://github.com/wso2/product-is/issues/14899
- Internal Issue: https://github.com/wso2-enterprise/wso2-iam-internal/issues/633

### CODE REVIEW

**Comment by lashinijay** [comment]
Reference PR: https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2124

File: components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpointTest.java [hunk 1]
```diff
@@ -84,8 +84,11 @@ public class JwksEndpointTest extends PowerMockIdentityBaseTest {
     private static final String ALG = "RS256";
     private static final String USE = "sig";
     private static final JSONArray X5C_ARRAY = new JSONArray();
-    private static final JSONArray X5T_ARRAY = new JSONArray();
-    private JwksEndpoint jwksEndpoint;
+    private static final String X5T = "YmUwN2EzOGI3ZTI0Y2NiNTNmZWFlZjI5MmVjZjdjZTYzZjI0M2MxNDQ1YjQwNjI3NjY" +
+            "yZmZlYzkwNzY0YjU4NQ";
+
+    private static final String rsa256Thumbprint = "be:07:a3:8b:7e:24:cc:b5:3f:ea:ef:29:2e:cf:7c:e6:3f:24:3c:" +
+            "14:45:b4:06:27:66:2f:fe:c9:07:64:b5:85";    private JwksEndpoint jwksEndpoint;
```
Comment (Thumimku) [inline components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpointTest.java:91]: Formatting

File: components/org.wso2.carbon.identity.oauth.endpoint/pom.xml [hunk 2]
```diff
@@ -346,6 +346,7 @@
                 <artifactId>spotbugs-maven-plugin</artifactId>
                 <configuration>
                     <threshold>High</threshold>
+                    <maxHeap>2048</maxHeap>
```
Comment (Thumimku) [inline components/org.wso2.carbon.identity.oauth.endpoint/pom.xml:349]: Refered from: https://stackoverflow.com/questions/57672501/using-findsecbugs-in-maven-throws-java-lang-outofmemoryerror/57980545#57980545

**Review by Thumimku — APPROVED** [review]
LGTM

**Pull request merged**